README.md

```diff
@@ -119,42 +97,41 @@ Example Usage:
 
 ```sh
 # Default - build Electron itself
-e build
+$ e build
 ```
 
 ```sh
 # Build the Electron binary and generates a dist zip file
-e build electron:dist
+$ e build electron:dist
```

Comment:
Adding this dollar means the commands can't be copy-pasted right? Not sure if that's worth the visual prettiness

Reply:
shell prompts removed 18bab5a27589c844eea2657532ce0f85657fe14a